peachjam/templates/peachjam/_gazette_search.html

```diff
@@ -0,0 +1,11 @@
+<form action="{% url 'search:search' %}" method="get" class="d-flex mb-3">
+  <input type="text"
+         class="form-control flex-grow-1"
+         autofocus
+         name="q"
+         placeholder="Search {{ locality.name }} {{ year }} Gazettes"/>
+  <input type="submit" class="btn btn-outline-primary ms-2" value="Search"/>
+  {% if year %}<input type="hidden" name="year" value="{{ year }}"/>{% endif %}
+  {% if locality %}<input type="hidden" name="locality" value="{{ locality }}"/>{% endif %}
+  <input type="hidden" name="doc_type" value="Gazette"/>
```

Comment:
@longhotsummer The gazette specific one facets on doc_type=Gazette. I could try adding doc_type facet to the generic one and adding the doc_type context to the listing pages.

Reply:
@longhotsummer Let me add that

Reply:
Changed :+1: